bin/extlvs.py

```diff
@@ -0,0 +1,69 @@
+# extlvs.py
```

Comment:
could you make sure this conform to pep-8?
https://peps.python.org/pep-0008/

Reply:
fixed